server/services/sync-user.ts

```diff
@@ -2,11 +2,13 @@ import { dbClient, collections } from '../db';
 import keycloak from '../keycloak';
 import logger from '../logger';
 
-export const syncUser = async ({ log }) => {
+export const syncUser = async ({ log }: { log: boolean }) => {
```

Comment:
That's not a return type, that's an argument type. Note the parenthesis. The return type is currently `Promise<any[]>`, since the return type of `updateDoc` on `USERS` is not defined.

Reply:
Definitely just misread that lol, thanks